src/content-tags-drawer/ContentTagsDropDownSelector.jsx

```diff
@@ -60,69 +95,62 @@ const ContentTagsDropDownSelector = ({
 
   return (
     <div style={{ marginLeft: `${level * 1 }rem` }}>
-      {tagPages.map((tagPage, pageNum) => (
-        // Array index represents the page number
-        // eslint-disable-next-line react/no-array-index-key
-        <React.Fragment key={`tag-page-${pageNum}`}>
-          {tagPage.isLoading ? (
-            <div className="d-flex justify-content-center align-items-center flex-row">
-              <Spinner
-                animation="border"
-                size="xl"
-                screenReaderText={intl.formatMessage(messages.loadingTagsDropdownMessage)}
-              />
-            </div>
-          ) : null }
-          {tagPage.isError ? 'Error...' : null /* TODO: show a proper error message */}
-
-          {tagPage.data?.map((tagData) => (
-            <React.Fragment key={tagData.value}>
-              <div
-                className="d-flex flex-row"
-                style={{
-                  minHeight: '44px',
-                }}
+      {tagPages.isLoading ? (
```

Comment:
Changed tagPages from and array to an object that consolidate all pages. More info in the hook